src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,256 @@
+
+-----------------------------------------------------------
+Material properties
+-----------------------------------------------------------
+
+The bulk and shear drained elastic moduli of the materials as well as its drained linear thermal expansion coefficient relating temperature change to stress and displacement changes are defined within the ``constitution`` tag as follows:
+ 
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ThermoElasticProperties -->
+  :end-before: <!-- SPHINX_ThermoElasticPropertiesEnd -->
+
+Here the solid density is also defined but it is not used because the gravitational effect is ignored in this example. To mimic a thermoelastic coupling without fluid flow, a negligible porosity and a zero Biot's coefficient are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PoroElasticProperties -->
+  :end-before: <!-- SPHINX_PoroElasticPropertiesEnd -->
+
+In this XML block, the Biot's coefficient is not directly defined but via the elastic bulk modulus :math:`K_{s}` of the solid skeleton as :math:`b_{Biot} = 1 - K/K_{s}`. In this example, we define a skeleton bulk modulus that is identical to the drained bulk modulus :math:`K` defined above to enforce the Biot's coefficient to zero.
+
+The thermal conductivities and the volumetric heat capacities of casing, cement and rock are defined by following XML blocks:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ThermalProperties -->
+  :end-before: <!-- SPHINX_ThermalPropertiesEnd -->
+
+and
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_HeatCapacityProperties -->
+  :end-before: <!-- SPHINX_HeatCapacityPropertiesEnd -->
+
+A negligible permeability is defined for all the three layers to simulate a thermoelastic problem without fluid flow.
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PermeabilityProperties -->
+  :end-before: <!-- SPHINX_PermeabilityPropertiesEnd -->
+
+Also, a neglibile volumetric heat capacity is defined for fluid to completely remove the thermal convection effect such that only thermal transfer via the diffusion phenomenon is considered.
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_FluidProperties -->
+  :end-before: <!-- SPHINX_FluidPropertiesEnd -->
+
+Other fluid properties such as viscosity, thermal expansion coefficient, etc. are not used because fluid flow is not simulated and pore pressure is zero everywhere.
```

Comment:
```suggestion
Other fluid properties such as viscosity, thermal expansion coefficient, etc. are not relevant to this example because fluid flow is ignored and pore pressure is zero everywhere.
```

Reply:
done